### ISSUE TITLE
Alternative model download website

### ISSUE THREAD

**Issue by gustavofuhr** [huoyijie/AdvancedEAST#40]
Can any of you guys please provide the model weights in a site for non-chinese speakers? ;-) pan.baidu is impossible for me to use.

**Comment by vinayakkailas** [comment]
HI Can you upload the pretrained model in google drive/drop box please?

**Comment by gfickel** [comment]
Apparently baidu now requires a login to download those files, so those browser extensions no longer work without an account. But I've found this tip on Quora explaining how to create one if your not in China: https://www.quora.com/How-do-I-sign-up-for-Baidu-without-a-Chinese-telephone-number

The chrome web translator helps a lot in the process.

With the user created I've used the baidu-dl chrome extension to get the link and download.

If the original code author authorizes I can upload it to another website and share with everyone, just let me know.

**Comment by hcnhatnam** [comment]
> Hei, I took the liberty to upload it on a public repo, right here: https://github.com/gfickel/advanced_east_weights
> 
> If the original author have any problem, just let me know. I will take it down without a problem.

hi @gfickel It's too hard to follow the instructions [here](https://www.quora.com/How-do-I-sign-up-for-Baidu-without-a-Chinese-telephone-number). Can you upload [tianchi ICPR dataset ](https://github.com/huoyijie/AdvancedEAST#training) in google drive/drop box please?Thank you.

**Comment by rupeshnehra** [comment]
@gustavofuhr @vinayakkailas Did you got any solution?
I am also unable to get dataset and understand model part

**Comment by jonatron** [comment]
I had files here, nobody mirrored them. I can't host them forever.
